File: JAVA/Program/Ibu.java
import java.util.ArrayList;
public class Ibu extends Orang{
    private String pekerjaan;
    private String hobi;
    
    //constructor
    public Ibu(){
    }   

    public Ibu(String nama, int usia, String jenisKelamin, String pekerjaan, String hobi){
        super(nama, usia, jenisKelamin);
        this.pekerjaan = pekerjaan;
        this.hobi = hobi;
    }

    //getter dan setter
    public String getPekerjaan(){
        return pekerjaan;
    }
    public void setPekerjaan(String pekerjaan){
        this.pekerjaan = pekerjaan;
    }
    public String getHobi(){
        return hobi;
    }
    public void setHobi(String hobi){
        this.hobi = hobi;
    }
    //menampilkan data ibu
    public void printIbu() {
    System.out.println("Nama          : " + getNama());
    System.out.println("Usia          : " + getUsia() + " tahun");
    System.out.println("Jenis Kelamin : " + getJenis());
    System.out.println("Pekerjaan     : " + getPekerjaan());
    System.out.println("Hobi          : " + getHobi());
}
}

File: PYTHON/Program/Ibu.py
from Orang import Orang

class Ibu(Orang):
    def __init__(self, Nama="", Usia=0, Jenis_kelamin="", Pekerjaan="", Hobi=""):
        super().__init__(Nama, Usia, Jenis_kelamin)
        self.Pekerjaan = Pekerjaan
        self.Hobi = Hobi
      

    # getter dan setter
    def getPekerjaan(self):
        return self.Pekerjaan

    def setPekerjaan(self, Pekerjaan):
        self.Pekerjaan = Pekerjaan

    def getHobi(self):
        return self.Hobi

    def setHobi(self, Hobi):
        self.Hobi = Hobi

    #untuk menampilkan data ibu
    def print_ibu(self):
        print(f"Nama          : {self.nama}")
        print(f"Usia          : {self.usia} tahun")
        print(f"Jenis Kelamin : {self.jenis_kelamin}")
        print(f"Pekerjaan     : {self.Pekerjaan}")
        print(f"Hobi          : {self.Hobi}")